spec/interactors/save_listing_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe SaveListing do
+  let(:service_area)    { create :service_area }
+  subject(:interaction) { SaveListing.run params }
+
+  describe 'validation' do
+    let(:params) {{
+      type: '',  # only Listing field with validation
```

Comment:
Do you mean where `person: {}` or is missing from params completely?

The interactor will reject that pretty early since there's no default configured for the person param; it won't even get to the `execute` method.

I tend to eschew specs that test the documented behavior of third-party libraries, but since we're new to `active_interaction`, there might be value in adding a spec demonstrating that behavior?

Reply:
Yeah, seems easy to leave off that param when dealing w raw json, but maybe what active_interaction provides is more integration w active_model ?